README.md

```diff
@@ -481,6 +481,67 @@ export const exampleImage = 'https://example.com/image.jpg'
 <img src={exampleImage} alt="Image alt text" />
 ```
 
+### Image Bundling
+
+With the [cwd](#cwd) and the remark plugin [remark-mdx-images](https://www.npmjs.com/package/remark-mdx-images)
+you can bundle images in your mdx!
+
+There are two loaders in esbuild that can be used here. The easiest is `dataurl`
+which outputs the images as inline data urls in the returned code.
+
+```js
+import {remarkMdxImages} from 'remark-mdx-images'
+
+const {code} = await bundleMDX(mdxSource, {
+  cwd: '/users/you/site/_content/pages',
+  xdmOptions: (vFile, options) => {
+    options.remarkPlugins = [remarkMdxImages]
+
+    return options
+  },
+  esbuildOptions: options => {
+    options.loader = {
+      ...options.loader,
+      '.png': 'dataurl'
+    }
+
+    return options
+  }
+})
+```
+
+The `file` loader requires a little more configuration to get working. With the
+`file` loader your images are copied to the output directory so esbuild needs
+to be set to write files and needs to know where to put them plus the url of
+the folder to be used in image sources.
+
+```js
+const {code} = await bundleMDX(mdxSource, {
+  cwd: '/users/you/site/_content/pages',
+  xdmOptions: (vFile, options) => {
+    options.remarkPlugins = [remarkMdxImages]
+
+    return options
+  },
+  esbuildOptions: options => {
+    // Set the `outdir` to your public directory.
+    options.outdir = '/users/you/site/public/img'
+    options.loader = {
+      ...options.loader,
+      // Tell esbuild to use the `file` loader for pngs
+      '.png': 'file'
+    }
+    // Set the public path to /img/ so image sources start /img/
+    options.publicPath = '/img/'
+
+    // Set write to true so that esbuild will output the files.
+    options.write = true
+
+    return options
+  }
+})
+```
+
```

Comment:
I am thinking this needs _for a more complete example see this post_ with a link to the article I'm writing about this feature.